### ISSUE TITLE
Feature request: Crown to serve as generic MIDI knob/button for musicians

### ISSUE THREAD

**Issue by unknowledgeable** [Logitech/logi_craft_sdk#16]
Hiya,

I'm a musician and have been using the Craft keyboard for a while and think a good use for the crown would be to turn it into a MIDI encoder and button. The crown feels so much better than any MIDI controllers' encoders I've come across and is always within reach. 

I'm thinking it would be amazing for it to have the functionality of one of the rotary encoders on a standard MIDI keyboard such as the ones on the top right of [this keyboard controller](http://www.akaipro.com/products/keyboard-controllers/mpk-249). Within Logitech Options, having the ability to reassign the CC parameter would be a massive bonus, the ability to change it from smooth turning and not would be pretty sweet and changing the sensitivity of the crown would be awesome (e.g. 720,360 or 180deg turns of the dial could be equivalent to 0-127 in MIDI).

I know I can use the crown as a scroll wheel which can do part of this but I find that in some audio software it behaves in the opposite way to expected and moves in either too fine or too broad increments. I also can't click around an audio plugin UI as I change a parameter with the knob.

Does this sound like it could be possible? And are people actively working on adding new features to the Craft?

Many thanks!

Tom

**Comment by reguar11** [comment]
Yes please !! I use Logic Pro X and I would love to use it as a Output Volume knob or to be able to assign and control modulation parameters 🙌🏻

Thank, Reinold.

**Comment by someofthem98** [comment]
It would be an absolute killer feature for our musicians!

**Comment by someofthem98** [comment]
Hi, there some update regarding support, if you want this features go here:
https://hi.computer/forum-community/talk-about-anything/logitech-craft-elephant/

**Comment by Flyweightt** [comment]
Hi I just got the Craft keyboard as well. 
Midi CC would be awesome.
For example you could control the master volume of your studio or any other parameters.

Can you achieve this with Autohotkey?
See: https://autohotkey.com/board/topic/72774-change-system-volume-via-midi-proof-of-concept/

Anyhow would love this feature Natively!

**Comment by clone18476** [comment]
Just got the Craft, it's all around a very solid keyboard, and the fact that we can make our own profiles is nice, but more customizeable features would be amazing, especially for working in DAW's, and even more so in Reaper (It's an open source DAW, free for anyone to use)

**Comment by Timoti** [comment]
Musicians should not have to become developers just to use this _potentially_ wonderful keyboard. The craft is one of the most expensive keyboards on the planet - surely support for common DAWs (esp logic BTW) should be baked in?

**Comment by clone18476** [comment]
Agreed, @Timoti !

**Comment by BendaCoding** [comment]
This would be super useful!

**Comment by reguar11** [comment]
> Musicians should not have to become developers just to use this _potentially_ wonderful keyboard. The craft is one of the most expensive keyboards on the planet - surely support for common DAWs (esp logic BTW) should be baked in?

Please tell you us you have some power to do that! I honestly bought it thinking it would be MIDI asignable or something like that and truth is that is very limited in it's capabilities :/. I still love it and would love to use the knob as a midi controller knob!!
